components/javascript-wrapper.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * This file contacts the common code used by all of the variations of any RDF
```

Comment:
Nope.  Deleted it for next check in.  It was a hangover from an earlier version.